uaclient/apt.py

```diff
@@ -104,6 +104,48 @@ def run_apt_command(
     except util.ProcessExecutionError as e:
         if "Could not get lock /var/lib/dpkg/lock" in str(e.stderr):
             error_msg += " Another process is running APT."
+
+        """
```

Comment:
Shouldn't this be an else clause as we shouldn't be able to run into an APT lock error and repository update errors right?